test/test_helper.exs

```diff
@@ -43,6 +43,22 @@ defmodule ElixirBenchWeb.TestHelpers do
     assert json_data == response
   end
 
+  defmacro assert_difference(queryable, value, do: expression) do
```

Comment:
this is interesting! :+1: I think it's a cool macro - generally macros should be our last resort. Instead of macros we could have also used anonymous functions.

However, especially for test helpers macros can be super helpful as they have the benefit over "usual" functions that they don't show up in the stack trace.

so yeah, seems good - but careful with macros :)

Reply:
Nice! :smile:  I couldn't think about a way to not evaluate the function before I count, so I needed the macro.